content/en/tracing/metrics/runtime_metrics/_index.md

```diff
@@ -163,6 +163,10 @@ Use the following environment variables to configure runtime metrics in your app
 : **Default**: `true` for Java, `false` for all other languages <br>
 **Description**: Enables the collection of runtime metrics. Metrics are sent to the Datadog agent, as configured for the instrumented application.
 
+`DD_RUNTIME_METRICS_RUNTIME_ID_ENABLED`
+: **Default**: `true` for Java, `false` for NodeJs, Ruby and Python. Does not exist for .NET and Go, runtime-id is always reported <br>
+**Description**: Enables enhanced runtime metrics, providing a `runtime_id` tag along with every metric. The `runtime_id` represents the application's process identifier and allows you to directly correlate runtime metrics with individual running applications.
```

Comment:
@bm1549 changed to this as .net and go just don't have the env var and it's not configurable, so itwont work if someone tries to deactivate it for these languages. Let me know if you think it's too complicated and we should just say instead:
> `true` for Java, .NET and Go, `false` for all other languages.